src/hydra-queue-runner/hydra-queue-runner.cc

```diff
@@ -854,6 +858,25 @@ int main(int argc, char * * argv)
     return handleExceptions(argv[0], [&]() {
         initNix();
 
+        /* Export a simple "up" metric, to allow monitoring that we're
+           still alive. */
+        std::thread([&]() {
+            prometheus::Exposer exposer{"127.0.0.1:8080"};
```

Comment:
I don't think we need to put this in a thread: exposer appears to make its own thread for serving metrics. I think we probably want to put the exposer in to the State struct, along with the various metrics we end up creating.